packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -102,20 +104,44 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
 
   const [currentMeetingState, setCurrentMeetingState] = useState<CallState>();
   const [currentPage, setCurrentPage] = useState<MeetingCompositePage>();
-  meetingAdapter.onStateChange((newState) => {
-    setCurrentPage(newState.page);
-    setCurrentMeetingState(newState.meeting?.state);
-  });
-
+  const [unreadChatMessages, setUnreadChatMessages] = useState<number>(0);
   const [showChat, setShowChat] = useState(false);
   const [showPeople, setShowPeople] = useState(false);
 
+  useEffect(() => {
+    const updateMeetingPage = (newState): void => {
+      setCurrentPage(newState.page);
+      setCurrentMeetingState(newState.meeting?.state);
+    };
+    meetingAdapter.onStateChange(updateMeetingPage);
+    return () => {
+      meetingAdapter.offStateChange(updateMeetingPage);
+    };
+  }, [meetingAdapter]);
+  useEffect(() => {
+    const incrementUnreadtChatMessages = (event: { message: ChatMessage }): void => {
```

Comment:
I think there is a mis-spelling here

Are we incrementing our unread chat messages or the number of unreadChatMessages. Again we want to be consistent about the naming to the type.

Reply:
okay I will revisit the naming here